python/setup.py

```diff
@@ -401,15 +528,10 @@ def append_cmake_bool(value, varname):
     def _bundle_arrow_cpp(self, build_prefix, build_lib):
         print(pjoin(build_lib, 'pyarrow'))
         move_shared_libs(build_prefix, build_lib, "arrow")
-        move_shared_libs(build_prefix, build_lib, "arrow_python")
         if self.with_cuda:
             move_shared_libs(build_prefix, build_lib, "arrow_cuda")
         if self.with_substrait:
             move_shared_libs(build_prefix, build_lib, "arrow_substrait")
-        if self.with_flight:
-            move_shared_libs(build_prefix, build_lib, "arrow_flight")
```

Comment:
Why is arrow_flight not needed anymore?

Reply:
Oh, that's a mistake (don't remember correctly but I think I have mistaken it with Python Flight module that wouldn't need to be moved in this part)